OCIContainer/doc/OCIContainerPlugin.md

```diff
@@ -115,6 +115,7 @@ Executes a command inside a running container. The path to the executable must r
 | :-------- | :-------- | :-------- |
 | result | object |  |
 | result.success | boolean | Whether the request succeeded |
+| result.error | string | Error information |
```

Comment:
Fix documentation to reflect Thunder schema meaning:

- errors array with items that consists mandatory error code and error message
- list all possible errors for this method